@xen-orchestra/lite/src/components/HostPatchesTable.vue

```diff
@@ -44,9 +44,13 @@ const props = defineProps<{
 }>();
 
 const sortedPatches = computed(() =>
-  [...props.patches].sort(
-    (patch1, patch2) => patch1.changelog.date - patch2.changelog.date
-  )
+  [...props.patches].sort((patch1, patch2) => {
+    if (patch1.changelog == null || patch2.changelog == null) {
+      return 0;
+    }
+
+    return patch1.changelog.date - patch2.changelog.date;
```

Comment:
Don't you think it would be better to have the patches without a changelog at the end of the array?

This way we ensure that we have a correct sort for all items that have a known changelog.